src/matter/cluster/BasicCluster.ts

```diff
@@ -4,17 +4,43 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { StringT, UnsignedIntT } from "../../codec/TlvObjectCodec";
+import { StringT, UnsignedIntT, BooleanT, Field, ObjectT } from "../../codec/TlvObjectCodec";
 import { AttributeSpec, ClusterSpec } from "./ClusterSpec";
 
+const CapabilityMinimaStructT = ObjectT({
```

Comment:
nit: "Struct" can be removed to be consistent with other type names
"T" is for template.